tests/tcl/tests/integration/slotmigrate.tcl

```diff
@@ -111,8 +111,8 @@ start_server {tags {"Src migration server"} overrides {cluster-enabled yes}} {
         set node1_id "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx01"
         $r1 clusterx setnodeid $node1_id
 
-        set cluster_nodes "$node0_id 127.0.0.1 $node0_host master - 0-10000"
```

Comment:
Here ?

Reply:
Yes. `127.0.0.1` is used here as the port number, because atoi is parsed to 127. I will replace atoi with parseint in another pr.